[DE] Add `People/Featured Artists` (#12487)
ppy/osu-wiki: wiki/People/Featured_Artists/de.md

---
tags:
  - FA
---

> # Gesponserte Künstler

Author: There is a financial backing because osu! buys a license from the musicians if I understand correctly.

If you don't like the term, I will change it to empfohlen.

> **Gesponserte Künstler** (auch *Featured Artists* oder kurz ***FA*** genannt) sind Musikschaffende, die mit osu! zusammenarbeiten, um einige ihrer Songs für die Verwendung in [Beatmaps](/wiki/Beatmap) und als Hintergrundmusik in den Livestreams des [osu! World Cup](/wiki/Tournaments#osu!-world-cup) zur Verfügung zu stellen.

Author: ```suggestion
**Gesponserte Künstler** (auch *Featured Artists* oder kurz ***FA*** genannt) sind Musiker, die mit osu! zusammenarbeiten, um einige ihrer Songs für die Verwendung in [Beatmaps](/wiki/Beatmap) und als Hintergrundmusik in den Livestreams des [osu! World Cup](/wiki/Tournaments#osu!-world-cup) zur Verfügung zu stellen.
```

Die [Auflistung der Featured Artists](https://osu.ppy.sh/beatmaps/artists) auf der Webseite präsentiert jeden einzelnen Künstler, der an dem Programm teilnimmt. Jede Künstlerseite enthält allgemeine Informationen über die Musik und eine Liste der Songs, die für die Verwendung in osu! lizenziert sind, zusammen mit vorkonfigurierten `.osz`-Dateien zur Erstellung von Beatmaps.

Das [osu!-Team](/wiki/People/osu!_team) arbeitet manchmal mit Künstlern zusammen, um neue Exklusivtitel für osu! zu produzieren, genannt [osu! originals](/wiki/osu!_originals). Diese werden als solche im FA-Katalog gekennzeichnet.

## Auswahl

> Neuzugänge zum Featured-Artist-Programm werden indirekt von der osu!-Community bestimmt. Wenn Mapper Beatmaps für einen Künstler entwerfen und ranken, spricht das osu!-Team den Künstler möglicherweise an, um auf das Featured-Artist-Programm aufmerksam zu machen. Gelegentlich werden Künstler jedoch basierend auf anderen Faktoren zum Featured-Artist-Programm eingeladen, wie beispielsweise ihre Präsenz in der Mapping-Community von osu!.

Author: I tried to avoid the verb `ranken` as much as possible, but it's not possible without loosing precision I think.

There is another occurence with `das/dem Ranking`.

> Neuzugänge zum Featured-Artist-Programm werden indirekt von der osu!-Community bestimmt. Wenn Mapper Beatmaps für einen Künstler entwerfen und ranken, spricht das osu!-Team den Künstler möglicherweise an, um auf das Featured-Artist-Programm aufmerksam zu machen. Gelegentlich werden Künstler jedoch basierend auf anderen Faktoren zum Featured-Artist-Programm eingeladen, wie beispielsweise ihre Präsenz in der Mapping-Community von osu!.

Author: ```suggestion
Neuzugänge zum Featured-Artist-Programm werden indirekt von der osu!-Community bestimmt. Wenn Mapper Beatmaps für einen Künstler entwerfen und ranken, spricht das osu!-Team den Künstler möglicherweise an, um auf das Featured-Artist-Programm aufmerksam zu machen. Gelegentlich werden Künstler jedoch basierend auf anderen Faktoren zum Featured-Artist-Programm eingeladen, wie beispielsweise ihren Bekanntheitsgrad in der Mapping-Community von osu!.
```

> Alle Songs im Featured-Artist-Programm werden nicht-exklusiv für die Verwendung in osu! lizenziert. Weitere Details für Musikschaffende, die dem Künstlerprogramm beitreten möchten, können im Artikel [Musiklizenzierung](/wiki/Legal/Music_licensing) nachgelesen werden.

Author: ```suggestion
Alle Songs im Featured-Artist-Programm werden nicht-exklusiv für die Verwendung in osu! lizenziert. Weitere Details für Musiker, die dem Künstlerprogramm beitreten möchten, können im Artikel [Musiklizenzierung](/wiki/Legal/Music_licensing) nachgelesen werden.
```

## Bekanntgabe neuer gesponserter Künstler

Einbindungen neuer gesponserter Künstler pro Jahr:

- **2016:** 8
- **2017:** 11
- **2018:** 12
- **2019:** 31
- **2020:** 65
- **2021:** 101
- **2022:** 104

> - **2023:** 64

Author: ```suggestion
- **2023:** 65
```

> - **2024:** *noch ausstehend*

Author: ```suggestion
- **2024:** 58
- **2025:** *noch ausstehend*
```

> Neue gesponserte Künstler werden wöchentlich im [Newsfeed](https://osu.ppy.sh/home/news) von osu! bekanntgegeben. Die meisten Ankündigungen enthalten eine Liste der lizenzierten Songs des Künstlers und einen Link zur Künstlerseite. Einige Ankündigungen sind auch mit Mapping-Wettbewerben oder von der [Mappers' Guild](/wiki/Community/Mappers_Guild) erstellten Beatmaps gepaart.

Author: We dont want to translate `Mapper's Guild`, do we?

Mappergilde/Mappinggilde 🤔

## Verwendung von der Community

Mapper sind dazu aufgerufen, aus Featured-Artist-Songs Beatmaps zu kreieren. Diese Songs werden mit voreingestelltem [Timing](/wiki/Client/Beatmap_editor/Timing) bereitgestellt und sind bereits für die Verwendung in osu! freigegeben, wodurch Nutzer sofort ohne Bedenken in das Mapping eintauchen können. Die in Arbeit befindliche Entwicklungsversion von osu!, genannt [osu!(lazer)](/wiki/Client/Release_stream/Lazer), präsentiert diese Beatmaps prominent im Spiel.

In allen Mapping-Wettbewerben, die in der [Wettbewerbsauflistung](https://osu.ppy.sh/community/contests) veröffentlicht werden, wie zum Beispiel dem [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) und der [Mapping Olympiad](/wiki/Contests/Mapping_Olympiad), müssen Teilnehmer Beatmaps zu Songs von gesponserten Künstlern erstellen. Einige von der Community organisierte Wettbewerbe verwenden diese Songs ebenfalls.

> Um das Mapping von Inhalten von gesponserten Künstlern zu werben, erstellten Mitglieder der Community die [Mappers' Guild](/wiki/Community/Mappers_Guild), innerhalb derer Mappern Aufgaben im Stil eines [MMOs](https://de.wikipedia.org/wiki/Massively_Multiplayer_Online_Game) gegeben wird, an denen sie mit anderen Mappern teilnehmen können. Für die Erstellung und das Ranking von Beatmaps werden Preise vergeben, unter anderem drei Stufen für Profilabzeichen. Mapper können der Gilde über die  [Webseite der Mappers' Guild](https://mappersguild.com/) beitreten. Seit [April 2023](https://osu.ppy.sh/home/news/2023-04-09-new-featured-artist-chon) können sich Mapper ebenfalls über alles, was mit der Erstellung von Beatmaps zu gesponserten Künstlern zusammenhängt, im Kanal `#featured-artists` des [osu! Discord-Servers](/wiki/Community/Discord_servers#official) austauschen.

Author: ```suggestion
Um das Mapping von Inhalten von gesponserten Künstlern zu werben, erstellten Mitglieder der Community die [Mappers' Guild](/wiki/Community/Mappers_Guild), innerhalb derer Mappern Aufgaben im Stil eines [MMOs](https://de.wikipedia.org/wiki/Massively_Multiplayer_Online_Game) gegeben wird, an denen sie mit anderen Mappern teilnehmen können. Für die Erstellung und das Ranking von Beatmaps werden Preise vergeben, unter anderem drei Stufen für Profilabzeichen. Mapper können der Gilde über die  [Webseite der Mappers' Guild](https://mappersguild.com/) beitreten. Seit [April 2023](https://osu.ppy.sh/home/news/2023-04-09-new-featured-artist-chon) können sich Mapper ebenfalls über alles, was mit der Erstellung von Beatmaps zu gesponserten Künstlern zusammenhängt, im Kanal `#featured-artists` des [osu! Discord-Servers](/wiki/Community/Discord_servers#offiziell) austauschen.
```

| ::{ flag=RU }:: [tieff](https://osu.ppy.sh/users/89619) | [tieff](https://osu.ppy.sh/beatmaps/artists/34) |
| ::{ flag=AU }:: [tomspicy](https://osu.ppy.sh/users/13210748) | [tomspicy](https://osu.ppy.sh/beatmaps/artists/437) |
| ::{ flag=HK }:: [Trainer LOLXD](https://osu.ppy.sh/users/14817071) | [Trina Lydia](https://osu.ppy.sh/beatmaps/artists/440) |
| ::{ flag=RU }:: [uselet](https://osu.ppy.sh/users/5256300) | [uselet](https://osu.ppy.sh/beatmaps/artists/402) |
| ::{ flag=CA }:: [VINXIS](https://osu.ppy.sh/users/4323406) | [VINXIS](https://osu.ppy.sh/beatmaps/artists/22) |
| ::{ flag=DE }:: [WangleLine](https://osu.ppy.sh/users/7994819) | [WangleLine](https://osu.ppy.sh/beatmaps/artists/427) |
| ::{ flag=DK }:: [water](https://osu.ppy.sh/users/11671515) | [watering](https://osu.ppy.sh/beatmaps/artists/416) |
| ::{ flag=KR }:: [WyvernP](https://osu.ppy.sh/users/3611701) | [WyvernP](https://osu.ppy.sh/beatmaps/artists/445) |
| ::{ flag=JP }:: [xorsizer](https://osu.ppy.sh/users/20466752) | [Symholic](https://osu.ppy.sh/beatmaps/artists/130) (Mitglied) |
| ::{ flag=TH }:: [YuugenP](https://osu.ppy.sh/users/2014016) | [rN](https://osu.ppy.sh/beatmaps/artists/190) |

> | ::{ flag=US }:: [Zenpakuwu](https://osu.ppy.sh/users/13226945) | [Zenpaku](https://osu.ppy.sh/beatmaps/artists/316) |

Author: ```suggestion
| ::{ flag=US }:: [Zenpakuwu](https://osu.ppy.sh/users/13226945) | [Zenpaku](https://osu.ppy.sh/beatmaps/artists/316) |
| ::{ flag=DE }:: [ZVLIAN](https://osu.ppy.sh/users/13687741) | [ZVLIAN](https://osu.ppy.sh/beatmaps/artists/455) |
```

## Verwendung von der Community

Mapper sind dazu aufgerufen, aus Featured-Artist-Songs Beatmaps zu kreieren. Diese Songs werden mit voreingestelltem [Timing](/wiki/Client/Beatmap_editor/Timing) bereitgestellt und sind bereits für die Verwendung in osu! freigegeben, wodurch Nutzer sofort ohne Bedenken in das Mapping eintauchen können. Die in Arbeit befindliche Entwicklungsversion von osu! namens [osu!(lazer)](/wiki/Client/Release_stream/Lazer) hebt solche Beatmaps deutlich hervor.

In allen Mapping-Wettbewerben, die in der [Wettbewerbsauflistung](https://osu.ppy.sh/community/contests) veröffentlicht werden, wie zum Beispiel dem [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) und der [Mapping Olympiad](/wiki/Contests/Mapping_Olympiad), müssen Teilnehmer Beatmaps zu Songs von gesponserten Künstlern erstellen. Einige von der Community organisierte Wettbewerbe verwenden diese Songs ebenfalls.

> Um das Mapping von Featured-Artist-Inhalten zu fördern, haben einige Mitglieder der Community die [Mappers' Guild](/wiki/Community/Mappers_Guild) gegründet. Innerhalb der "Gilde" werden Mappern Aufgaben im Stil eines [MMOs](https://de.wikipedia.org/wiki/Massively_Multiplayer_Online_Game) gegeben, an denen sie mit anderen Mappern teilnehmen können. Für die Erstellung und das Ranking von Beatmaps werden Preise vergeben, unter anderem drei Stufen für Profilabzeichen. Mapper können der Gilde über die  [Webseite der Mappers' Guild](https://mappersguild.com/) beitreten. Seit [April 2023](https://osu.ppy.sh/home/news/2023-04-09-new-featured-artist-chon) können sich Mapper ebenfalls über alles, was mit der Erstellung von Beatmaps zu gesponserten Künstlern zusammenhängt, im Kanal `#featured-artists` des [osu! Discord-Servers](/wiki/Community/Discord_servers#offiziell) austauschen.

Author: ```suggestion
Um das Mapping von Featured-Artist-Inhalten zu fördern, haben einige Mitglieder der Community die [Mappers' Guild](/wiki/Community/Mappers_Guild) gegründet. Innerhalb der "Gilde" werden Mappern Aufgaben im Stil eines [MMOs](https://de.wikipedia.org/wiki/Massively_Multiplayer_Online_Game) gegeben, an denen sie mit anderen Mappern teilnehmen können. Für die Erstellung und das Ranking von Beatmaps werden Preise vergeben, unter anderem drei Stufen an Profilabzeichen. Mapper können der Gilde über die [Webseite der Mappers' Guild](https://mappersguild.com/) beitreten. Seit [April 2023](https://osu.ppy.sh/home/news/2023-04-09-new-featured-artist-chon) können sich Mapper ebenfalls über alles, was mit der Erstellung von Beatmaps zu gesponserten Künstlern zusammenhängt, im Kanal `#featured-artists` des [osu! Discord-Servers](/wiki/Community/Discord_servers#offiziell) austauschen.
```
